### ISSUE TITLE
Can't keep the cursor visible

### ISSUE THREAD

**Issue by Nicaullait** [MohamedRejeb/Compose-Rich-Editor#176]
Hello

I'm using your component inside a ComposeView which is inside a NestedScrollView (the NestedScrollView is here because i was unable to scroll inside de component if the page was scrollable)
When i'm writing a new line, i can't move the content up because i cant find the cursor's position in the editor.
Same issue when I'm editing the middle of the text.

Maybe you know a workaround or can you expose the cursor's position ?

Thanks

**Comment by MohamedRejeb** [comment]
Hi,
If you use the Compose `TextField` do you have the same issue or not? Because it looks like an issue related to Compose `TextField` not the library.

**Comment by Nicaullait** [comment]
You are right, the issue exist on the TextField. But there is a workaround using the cursor offset like [this one on Stackoverflow](https://stackoverflow.com/questions/77365585/textfield-scroll-to-cursor).

**Comment by MohamedRejeb** [comment]
The `BasicRichTextEditor` is built on top of `BasicTextField` so you should be able to use this workaround with the rich editor as well.

**Comment by Goooler** [comment]
Seems it has been fixed in TextField2, I guess this will be resolved after migrating to Compose 1.7.

**Comment by MohamedRejeb** [comment]
Yes, BasicTextField2 is not ready yet to be used with a rich text library because it still doesn't support annotated string.
But we need it to fix a lot of things and get rid of some limitations.
Currently, I'm planning to copy BTF2 code into this library and maintain my own version so I can be able to add the missing features.
